bridge/test/utils.js

```diff
@@ -63,6 +65,29 @@ function checkRcpt(tx) {
     checkGas(tx.receipt.gasUsed);
   }
 
+// @param tx is an object returned from the smartcontract tx {
+//  tx: @string,
+//  receipt: {
+//    gasUsed: @number
+//  }
+// }
+// @return BN the gas used multiplied by the gas price
+const getGasUsed = async (tx) => {
```

Comment:
awesome util!

Reply:
maybe I'd rename it to `getGasUsedByTx` or `getGasUsedByTransaction`?

Reply:
getGasUsedByTx, agree! ;D